README.md

```diff
@@ -7,25 +7,25 @@ the IPTB build in this project.
 ### Example
 
 ```
-$ iptb auto -type localipfs -count 5
+$ iptb auto -type localbtfs -count 5
 <output removed>
 
 $ iptb start
 
 $ iptb shell 0
-$ echo $IPFS_PATH
+$ echo $BTFS_PATH
 /home/iptb/testbed/testbeds/default/0
 
-$ echo 'hey!' | ipfs add -q
-QmNqugRcYjwh9pEQUK7MLuxvLjxDNZL1DH8PJJgWtQXxuF
+$ echo 'pizza!' | btfs add -q
+QmZio6HUGdQfPq1tMNGUyDvcGnBERwyBdic75bXXU5txcj
 
 $ exit
 
 $ iptb connect 0 4
 
 $ iptb shell 4
-$ ipfs cat QmNqugRcYjwh9pEQUK7MLuxvLjxDNZL1DH8PJJgWtQXxuF
-hey!
+$ btfs cat QmNqugRcYjwh9pEQUK7MLuxvLjxDNZL1DH8PJJgWtQXxuF
```

Comment:
Here, the btfs add has been changed to the new QmZio6HUGdQfPq1tMNGUyDvcGnBERwyBdic75bXXU5txcj.
Do you want to change the btfs cat Hash as well?

Reply:
good catch, will fix.